ElectronClient/gui/KeymapConfigScreen.tsx

```diff
@@ -0,0 +1,229 @@
+const { _ } = require('lib/locale.js');
+// const { bridge } = require('electron').remote.require('./bridge');
+// const Setting = require('lib/models/Setting');
+
+interface KeymapItem {
+	label: string,
+	accelerator: string,
+	command: string
+	isEditing: boolean,
+}
+
+// Temporary values
+const commandNames = [
+	'textCopy',
+	'textCut',
+	'textPaste',
+];
+const getAccelerator = (commandName: string) => {
+	switch (commandName) {
+	case 'textCopy':
+		return 'Ctrl+C';
+	case 'textCut':
+		return 'Ctrl+X';
+	case 'textPaste':
+		return 'Ctrl+V';
+	default:
+		throw new Error('Invalid command');
+	}
+};
+
+const initialKeymap = commandNames.map(commandName => {
+	return {
+		label: CommandService.instance().label(commandName),
+		accelerator: getAccelerator(commandName),
+		command: commandName,
+		isEditing: false,
+	};
+});
+
+const KeymapConfigScreen = (props: { theme: Object }) => {
+	const [keymap, setKeymap] = React.useState([...initialKeymap]);
+	const [filter, setFilter] = React.useState('');
+
+	const theme = themeStyle(props.theme);
+	const containerStyle = {
+		...theme.containerStyle,
+		padding: 10,
+		overflow: 'auto',
+	};
+
+	const updateAccelerator = (commandName: string, accelerator: string) => {
+		const _keymap = [...keymap];
+
+		_keymap.find(item => item.command === commandName).accelerator = accelerator;
+		_keymap.find(item => item.command === commandName).isEditing = false;
+		setKeymap(_keymap);
+	};
+
+	const updateIsEditing = (commandName: string, isEditing: boolean) => {
+		const _keymap = [...keymap];
+
+		_keymap.forEach(item => item.isEditing = false);
+		_keymap.find(item => item.command === commandName).isEditing = isEditing;
+		setKeymap(_keymap);
+	};
+
+	return (
+		<div style={containerStyle}>
+			<TopActions
+				theme={props.theme}
+				filter={filter}
+				setFilter={setFilter}
+			/>
+			<KeymapTable
+				theme={props.theme}
+				keymap={keymap}
+				updateAccelerator={updateAccelerator}
+				updateIsEditing={updateIsEditing}
+			/>
+		</div>
+	);
+};
+
+const TopActions = (props: {
+	theme: any,
+	filter: string,
+	setFilter: (filter: string) => void
+}) => {
+	const theme = themeStyle(props.theme);
+	const topActionsStyle = {
+		...theme.containerStyle,
+		marginTop: 10,
+		display: 'flex',
+		flexDirection: 'row',
+	};
+	const inlineButtonStyle = {
+		...theme.buttonStyle,
+		height: theme.inputStyle.height,
+		padding: 0,
+		marginLeft: 12,
+	};
+	const filterInputStyle = {
+		...theme.inputStyle,
+		flexGrow: 1,
+	};
+
+	return (
+		<div style={topActionsStyle}>
+			<input
+				style={filterInputStyle}
+				placeholder={_('Search..')}
+				value={props.filter}
+				onChange={e => props.setFilter(e.target.value)}
+			/>
+
+			<button style={inlineButtonStyle} onClick={() => console.log()}>
+				{_('Import')}
+			</button>
+			<button style={inlineButtonStyle} onClick={() => console.log()}>
+				{_('Export')}
+			</button>
+		</div>
+	);
+};
+
+const KeymapTable = (props: {
+	theme: any,
```

Comment:
Please don't use `any` when the property has a specific type, in this case `number`. Also likewise all props should be defined in an interface.